collect_app/src/main/java/org/odk/collect/android/activities/InstanceUploaderActivity.java

```diff
@@ -1,5 +1,4 @@
 /*
- * Copyright (C) 2009 University of Washington
```

Comment:
This should stay. You can add an `@author` line to indicate the copyright for your additions. Also, commit history is used to identify ownership.

Reply:
Sorry, it seems I removed it by mistake. Thanks for that